bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -352,4 +368,25 @@
 		<description>The level of the heating</description>
 		<state max="100" min="0" step="1" pattern="%d %%"/>
 	</channel-type>
+
+	<channel-type id="current_heating_mode">
+		<item-type>String</item-type>
+		<label>Current Heating Mode</label>
+		<description>Current heating mode of the Somfy thermostatic valve</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="open_closed_valve">
+		<item-type>String</item-type>
+		<label>Valve Open/Closed state</label>
+		<description>Current open/closed state of the Somfy thermostatic valve</description>
+		<state readOnly="true"/>
+	</channel-type>
```

Comment:
If this has only open and closed, you could make it `Contact`, which has exactly these states.

Reply:
yeah, good idea. fixes